cli/cmd/cloud_account.go

```diff
@@ -12,9 +14,75 @@ var (
 		Short:   "Manage cloud accounts",
 		Long:    "Manage cloud account integrations with Lacework",
 	}
+
+	// used by cloud account list to list only a single type of cloud account
+	cloudAccountType string
+
+	// cloudAccountsListCmd represents the list sub-command inside the cloud accounts command
+	cloudAccountListCmd = &cobra.Command{
+		Use:   "list",
+		Short: "List all available cloud account integrations",
+		Args:  cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			var (
+				cloudAccounts api.CloudAccountsResponse
+				err           error
+			)
+
+			if cloudAccountType != "" {
```

Comment:
At this point, how is `cloudAccountType` anything other than `""1

Reply:
I believe this is only called when the command is run

Reply:
And this then sets the value of the var
```
	// add type flag to cloud accounts list command
	cloudAccountListCmd.Flags().StringVarP(&cloudAccountType,
		"type", "t", "", "list all cloud accounts of a specific type",
	)
```

Reply:
I missed this flag.  TY.